packages/composables/src/useProductAssociations.ts

```diff
@@ -59,10 +62,10 @@ export function useProductAssociations(
         const response = await invokeGet(
           {
             address: `${getProductDetailsEndpoint(
-              product.value.id
+              product.value.id,
```

Comment:
is the TS okay with that? `getProductDetailsEndpoint` should take 1 param, and we're passing second `undefined` param here by accident